testing/tests/ui/filter-signature-validation.stderr

```diff
@@ -0,0 +1,77 @@
+error: Filter function missing required input and environment args
```

Comment:
The error could be better: let's show what is expected too:

```
error: Filter function missing required input and environment args. Example: `fn filter0(_: &dyn std::fmt::Display, _: &dyn askama::Values) -> askama::Result<String>`
```

It would be so great if we had access to the span API to add error messages... T_T